docs/standard/glossary.md

```diff
@@ -270,6 +270,16 @@ Legacy .NET documentation sometimes uses ".NET platform" to mean either an [impl
 
 "Platform" has a different meaning in the phrase "developer platform," which refers to software that provides tools and libraries for building and running apps. .NET is a cross-platform, open-source developer platform for building many different types of applications.
 
+## poco
+
+A poco&mdash;or a plain old class/[CLR](#clr) object&mdash;is a C# object that contains only public properties or fields. A poco shouldn't contain any other members, such as:
```

Comment:
Outside of this wikipedia article, I've always heard it explained as "CLR object", which makes more sense to me than "class object", especially given the etymology indicating it came from Plain Old Java Object.

Also, I believe it's OK to refer to POCOs of .NET languages other than C#, which would be another reason to prefer CLR rather than "class".

```suggestion
A POCO&mdash;or a plain old [CLR](#clr) object&mdash;is an object that contains only public properties or fields. A POCO shouldn't contain any other members, such as:
```

Reply:
Before I saw the Wiki I'd never heard of it being called CLR object. I left it as both, but I've learned it as "class object".